python/ray/data/tests/test_operator_fusion.py

```diff
@@ -259,7 +259,8 @@ def test_read_with_map_batches_fused_successfully(
 
     # Test that fusion of map operators merges their block sizes in the expected way
     # (taking the max).
-    ds = ray.data.read_parquet(temp_dir)
+    n = 10
+    ds = ray.data.range(n)
```

Comment:
Nit: Minimally passing tests is a good practice because it can make the intent of the test more clear.
```suggestion
    ds = ray.data.range(1)
```